sources/lists/dns-bh.go

```diff
@@ -9,7 +9,7 @@ import (
 
 //NewDNSBHList returns a new DNSBHList object
 func NewDNSBHList() list.List {
-	return NewLineSeperatedList(
+	return NewLineSeparatedList(
```

Comment:
"Seperated" => "Separated" was changed throughout the project. I will also make this change in a pull request in RITA once this goes through. But it is safe to accept the change here first since RITA is pinned to a specific version of this code and we will have to explicitly tell it to use a different version.